_includes/footer.html

```diff
@@ -24,3 +24,10 @@
     </div>
   </div>
 </div>
+<script>
+  // TODO: only show on hover. This is broken on my computer.
+  anchors.options = {
+    visible: 'touch',
+  };
+  anchors.add('h2');
```

Comment:
they make links to every h2 automatically so they can be linked in the navbar. they are kinda ugly tho so I can remove the link icon next to the heading itself